service/package/install_translations.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+# a helper script for compiling and installing the translations
+
+PODIR=$(mktemp --directory --suffix "-agama-po")
+
+tar xfjv "$1" -C "$PODIR"
+find "$PODIR" -name "*.po" -exec sh -c 'L=`basename "{}" .po` && mkdir -p "$RPM_BUILD_ROOT/usr/share/YaST2/locale/$L/LC_MESSAGES" && msgfmt -o "$RPM_BUILD_ROOT/usr/share/YaST2/locale/$L/LC_MESSAGES/agama.mo" "{}"' \;
```

Comment:
Note: Originally I wanted to put this line into the `gem2rpm.yml` file but I fell into the escaping hell (shell script in a `find` argument which is another shell script placed in an YAML file... :scream: ). So I put that into a separate script to make reading and writing easier.